test/module.mk

```diff
@@ -456,22 +457,7 @@ TEST_SRC += \
 		$(DIR)/test_SM_higgs_loop_corrections.cpp \
 		$(DIR)/test_SM_tree_level_spectrum.cpp \
 		$(DIR)/test_SM_three_loop_spectrum.cpp \
-		$(DIR)/test_SM_two_loop_spectrum.cpp \
```

Comment:
Why has this test been removed?

Reply:
There is no reason to remove it. Undone in aa5b527.